src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,129 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidInputDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String LOGIN_EXISTED_IN_STORAGE = "unknownCactus";
+    private static final String DEFAULT_LOGIN = "user1";
+    private static final String DEFAULT_PASSWORD = "user111";
+    private static final int DEFAULT_AGE = 18;
+    private static final String WRONG_PASSWORD = "12345";
+    private static final int WRONG_AGE = 15;
+    private static final String EXCEPTION = InvalidInputDataException.class.toString();
+
+    private static RegistrationServiceImpl registrationService;
+    private static StorageDaoImpl storageDao;
+    private static User actual;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        storageDao = new StorageDaoImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        User expectedUser = new User(LOGIN_EXISTED_IN_STORAGE, DEFAULT_PASSWORD, DEFAULT_AGE);
+        storageDao.add(expectedUser);
+    }
+
+    @Test
+    void register_loginNull_NotOk() {

```

Comment:
```suggestion
    void register_loginNull_notOk() {
```
the same for other methods